vispy/scene/node.py

```diff
@@ -36,7 +36,7 @@ class Node(object):
     # Needed to allow subclasses to repr() themselves before Node.__init__()
     _name = None
 
-    def __init__(self, parent=None, name=None, **kwargs):
+    def __init__(self, parent=None, name=None):
```

Comment:
This is the "endpoint" for the `**kwarg` passing, so it was dangerous for us to still have it here. If the kwargs haven't been consumed by this point, we should throw an error.